internal/handlers/sqlite/msg_create.go

```diff
@@ -78,6 +90,22 @@ func (h *Handler) MsgCreate(ctx context.Context, msg *wire.OpMsg) (*wire.OpMsg,
 		return nil, err
 	}
 
+	if strings.HasPrefix(collectionName, ".") {
+		msg := fmt.Sprintf("Collection names cannot start with '.': %s", collectionName)
+		return nil, commonerrors.NewCommandErrorMsgWithArgument(commonerrors.ErrInvalidNamespace, msg, "create")
+	}
+
+	if !validateCollectionNameRe.MatchString(collectionName) ||
+		!utf8.ValidString(collectionName) {
+		msg := fmt.Sprintf("Invalid collection name: '%s.%s'", dbName, collectionName)
+		return nil, commonerrors.NewCommandErrorMsgWithArgument(commonerrors.ErrInvalidNamespace, msg, "create")
+	}
+
+	if strings.HasPrefix(collectionName, reservedPrefix) {
+		msg := fmt.Sprintf("Invalid collection name (%q is a reserved prefix): '%s.%s'", reservedPrefix, dbName, collectionName)
```

Comment:
You might want to return the error message the same as other cases `Invalid collection name: '%s.%s'`. When we do run dance diff test for sqlite, we would want this test to pass

https://github.com/FerretDB/dance/blob/main/tests/diff/names_test.go#L48

Reply:
Any comment?

Reply:
Sorry, missed this one, yeah makes sense